end-to-end-tests/README.md

```diff
@@ -0,0 +1,47 @@
+# End-to-End tests with robotframework
+
+
+
+## Installation guide
+This installation guide works atleast for Windows 10, not sure yet if it works for other OS:s since no one has tested with other OS yet. It should work, but who knows.
```

Comment:
Tän rivin voi varmaan muotoilla uudelleen esim.: These instructions have been tested to work with Unix and Windows 10.

"I am using python 3.8." voisi muotoilla esim. "The tests have been run with python 3.8."